src/essentia/essentiamath.h

```diff
@@ -738,8 +738,88 @@ inline Real hz2hz(Real hz){
   return hz;
 }
 
-inline Real hz2cents(Real hz) {
-  return 12 * std::log(hz/440)/std::log(2.) + 69;
+inline Real cents2hz(Real frequencyB, Real cents) {
+  return frequencyB * powf(2.0, cents / 1200.0);
+}
+
+inline Real hz2cents(Real frequencyA, Real frequencyB) {
+  return 1200 * log2(frequencyA / frequencyB);
```

Comment:
`frequencyA` vs `frequencyB` can be unclear to users. Maybe we could use a different naming for `frequencyB` like `referenceFrequency`?

Reply:
sure, that makes sense. I doubted with `tuningFrequency` but `referenceFrequency` works better. Thanks.